libs/application-generic/src/usecases/get-subscriber-global-preference-v1/get-subscriber-global-preference-v1.usecase.ts

```diff
@@ -0,0 +1,101 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
Compare this new, simplified use-case to the [previously consumed use-case](https://github.com/novuhq/novu/blob/add-simple-global-preference-use-case/libs/application-generic/src/usecases/get-subscriber-global-preference/get-subscriber-global-preference.usecase.ts) - that previously consumed use-case that is now only used for fetching Inbox preferences.